From ad6272d7132f778a5a60d54411eb4048a5f15146 Mon Sep 17 00:00:00 2001
From: Brock Noland <brock@apache.org>
Date: Sat, 31 Jan 2015 19:05:56 +0000
Subject: [PATCH 0010/1164] HIVE-9526 - ClassCastException thrown by
 HiveStatement (Ashish Singh via Brock)

git-svn-id: https://svn.apache.org/repos/asf/hive/trunk@1656238 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 90a9cce1e9853347d73c0bae1a30a49a31f106c1)
---
 .../java/org/apache/hive/jdbc/HiveStatement.java   |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/jdbc/src/java/org/apache/hive/jdbc/HiveStatement.java b/jdbc/src/java/org/apache/hive/jdbc/HiveStatement.java
index d8e33d3..c2d0c9d 100644
--- a/jdbc/src/java/org/apache/hive/jdbc/HiveStatement.java
+++ b/jdbc/src/java/org/apache/hive/jdbc/HiveStatement.java
@@ -838,7 +838,7 @@ public boolean hasMoreLogs() {
     RowSet rowSet = RowSetFactory.create(tFetchResultsResp.getResults(),
         connection.getProtocol());
     for (Object[] row : rowSet) {
-      logs.add((String)row[0]);
+      logs.add(String.valueOf(row[0]));
     }
     return logs;
   }
-- 
1.7.9.5

